src/lib/apiTypesV2/Registration.h

```diff
@@ -31,54 +31,15 @@
 #include "apiTypesV2/EntID.h"
 
 
-#if 0
```

Comment:
@kzangeli I understand this could be cleaned out, right?

Reply:
Yes

Reply:
NTC